.github/workflows/CI.yml

```
@@ -0,0 +1,32 @@
+name: CI
+
+# Trigger the workflow on push or pull request
+on:
+  - push
+  - pull_request
+
+jobs:
+  test:
+    name: "Test: ${{ matrix.gap-branch }} on ${{ matrix.os }}"
```

Perhaps this would avoid potential confusion?
```suggestion
    name: "Test: GAP ${{ matrix.gap-branch }} on ${{ matrix.os }}"
```

---

I thought I'd already done that! Thanks.